back-end/e-learning/database/migrations/2022_02_21_061923_create_words_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('words', function (Blueprint $table) {
+            $table->id();
+            $table->integer('category_id')->unsigned();
+            $table->string('word');
```

Comment:
better if this is not word, coz the table/object name is already `word`, it would look like `$word->word`. you can use `value` or `content`, so it would look like `$word->value` or `$word->content`